### PR TITLE
fix: position UI at the top layer

### PR DESCRIPTION
**Pull request by ValeraS**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [ ] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

- [ ] Closes currently open issue: Closes #XXX

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ const filterStack = stackTraceFilter();
 // Wrapper for shadow DOM
 const testDiv = document.createElement('div');
 testDiv.setAttribute('id', 'fcc_test_suite_wrapper');
+// position our user interface at the top layer
+testDiv.style.setProperty('position', 'relative', 'important');
+testDiv.style.setProperty('z-index', '99999', 'important');
```
Comment (tbushman) [inline src/index.js]: Does it work if these styles are placed in one of the CSS files?

Comment (ValeraS) [reply src/index.js]: @tbushman It doesn't work because of the shadowDOM

**Comment by tbushman** [comment]
LGTM!

**Comment by ValeraS** [comment]
@tbushman Did errors appear in this merge in Travis CI? Is `CDN_USER` and `CDN_API_KEY` set?

**Comment by tbushman** [comment]
@ValeraS No errors. Good point. Our environment variables are named otherwise. We need to change `CDN_USER` to `GITHUB_USER` and change `CDN_API_KEY` to `GITHUB_API_KEY` . I'd do it, but I'm sure you have a way you want it so go for it.

**Comment by ValeraS** [comment]
@tbushman `GITHUB_USER` and `GITHUB_API_KEY` is used to commit in this repo, for commit in CDN repo we need another user and key.

**Pull request merged**